### PR TITLE
Scalar results from SQL repr nicely

### PR DESCRIPTION
**Pull request by mrocklin**
The solution feels a bit hacky.

### CODE REVIEW

File: blaze/interactive.py [hunk 1]
```diff
@@ -224,7 +233,19 @@ def expr_repr(expr, n=10):
 
     # Scalars
     if ndim(expr) == 0 and isscalar(expr.dshape):
-        return repr(compute(expr))
+        result = compute(expr)
+        if 'float' in str(expr.dshape):
```
Comment (cpcloud) [inline blaze/interactive.py]: i think there's a stronger way to check this:

```python
expr.dshape.measure in datashape.floating.types
```

Comment (mrocklin) [inline blaze/interactive.py]: Hrm, didn't know that those existed.  What about option types?

Comment (cpcloud) [inline blaze/interactive.py]:
```python
getattr(expr.dshape.measure, 'ty', expr.dshape.measure) in datashape.floating.types
```

Comment (mrocklin) [inline blaze/interactive.py]: I'm actually tempted to stick with `'float' in str(expr.dshape)`.  Given that we're under the `isscalar` check this is actually decently strong.  It's also fairly clear.

**Comment by mrocklin** [comment]
I apologize for merging this.  Feeling time pressured.

**Comment by cpcloud** [comment]
no problem, was really just a suggestion

**Pull request merged**